contents/careers/designer.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Designer
+sidebar: Careers
+showTitle: true
+---
+
+"Design is not just what it looks like and feels like. Design is how it works." - Steve Jobs, co-founder of [Apple](https://apple.com).
```

Comment:
I like the quote but using steve jobs in a design thing might be too cliché and turn off people

Reply:
I'll find another that has the same sentiment. I was half wondering if this would be a bit of a stereotypical "nightmare startup founder" thing to quote from, but I love this point!

Reply:
Fixed